src/button/button-group.scss

```diff
@@ -3,7 +3,7 @@
 @import '../style/index';
 
 @mixin iui-button-group {
-  display: inline-flex;
+  display: flex;
```

Comment:
Ugh, I think `inline-flex` was intentional that it would take as much place as needed.

Reply:
inline sucks though

Reply:
In your case, yes 😀 But for the user it doesn't take whole width, only as much as needed.